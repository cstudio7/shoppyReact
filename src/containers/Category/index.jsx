import React from 'react';
import Navbar from '../Navbar';
import CategoryComponent from '../../components/InfoCard';

export default function Category() {
  return (
    <Navbar
      displayPage={(
        <CategoryComponent
          nameToDisplay="categoryName"
          imageUrl="https://i.imgur.com/UBDlxxg.jpg"
          currentPage="category"
          description="this is a category description"
          userRole="admin"
        />
)}
    />
  );
}
