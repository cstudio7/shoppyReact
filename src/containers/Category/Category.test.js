import React from 'react';
import { shallow } from 'enzyme';
import Category from './index';

describe('Test the Category Component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Category />);
    expect(wrapper).toMatchSnapshot();
  });
});
