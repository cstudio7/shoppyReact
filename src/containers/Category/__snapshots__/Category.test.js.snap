// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the Category Component should match snapshot 1`] = `
<Navbar
  displayPage={
    <InfoCards
      currentPage="category"
      description="this is a category description"
      imageUrl="https://i.imgur.com/UBDlxxg.jpg"
      mobileNumber=""
      nameToDisplay="categoryName"
      userRole="admin"
    />
  }
/>
`;
